rlp/rlpgen/gen_test.go

```diff
@@ -47,7 +47,7 @@ func init() {
 	}
 }
 
-var tests = []string{"uints", "nil", "rawvalue", "optional", "bigint", "uint256", "alias"}
+var tests = []string{"uints", "nil", "rawvalue", "optional", "bigint", "uint256", "alias", "named"}
```

Comment:
```suggestion
var tests = []string{"uints", "nil", "rawvalue", "optional", "bigint", "uint256", "alias", "named.libevm"}
```

This will allow the new files to have the `.libevm` pattern in them too, which makes it easier to differentiate new from modified code.